docker-compose-dev.yml

```diff
@@ -1,6 +1,7 @@
 version: "3.3"
 services:
   mongo:
+    container_name: mongo
     image: mongo:latest
```

Comment:
Could you please specidy exact version, that you are using. I mean something like `4.4.11` or `5.0.5`